catalyst/callbacks/metrics/__init__.py

```diff
@@ -29,3 +29,5 @@
     IOUCallback,
     TrevskyCallback,
 )
+
+from catalyst.callbacks.metrics.scikit_learn import SklearnCallback
```

Comment:
let's import that callback only if `sklearn` is available in extensions
here is a good PR as example https://github.com/catalyst-team/catalyst/pull/1196

Reply:
done